src/ai/backend/client/func/user.py

```diff
@@ -259,6 +261,7 @@ async def create(
         totp_activated: bool = False,
         group_ids: Iterable[str] = None,
         fields: Iterable[FieldSpec | str] = None,
+        enable_sudo_session: bool = False,
```

Comment:
```suggestion
        enable_sudo_session: bool = False,
        fields: Iterable[FieldSpec | str] = None,
```

I think it's better to receive the `fields` param at the end like other files.